{"variants":[{"paths":["\/documentation\/facecapture\/facecaptureerror\/localizederror-implementations"],"traits":[{"interfaceLanguage":"swift"}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"identifier":{"url":"doc:\/\/FaceCapture\/documentation\/FaceCapture\/FaceCaptureError\/LocalizedError-Implementations","interfaceLanguage":"swift"},"topicSections":[{"title":"Instance Properties","identifiers":["doc:\/\/FaceCapture\/documentation\/FaceCapture\/FaceCaptureError\/failureReason","doc:\/\/FaceCapture\/documentation\/FaceCapture\/FaceCaptureError\/helpAnchor","doc:\/\/FaceCapture\/documentation\/FaceCapture\/FaceCaptureError\/recoverySuggestion"],"generated":true}],"kind":"article","metadata":{"modules":[{"name":"FaceCapture"}],"role":"collectionGroup","title":"LocalizedError Implementations"},"hierarchy":{"paths":[["doc:\/\/FaceCapture\/documentation\/FaceCapture","doc:\/\/FaceCapture\/documentation\/FaceCapture\/FaceCaptureError"]]},"references":{"doc://FaceCapture/documentation/FaceCapture":{"role":"collection","title":"FaceCapture","abstract":[{"type":"text","text":"Capture faces using your deviceâ€™s camera"}],"identifier":"doc:\/\/FaceCapture\/documentation\/FaceCapture","kind":"symbol","type":"topic","url":"\/documentation\/facecapture"},"doc://FaceCapture/documentation/FaceCapture/FaceCaptureError/helpAnchor":{"role":"symbol","title":"helpAnchor","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"helpAnchor"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}],"abstract":[],"identifier":"doc:\/\/FaceCapture\/documentation\/FaceCapture\/FaceCaptureError\/helpAnchor","kind":"symbol","type":"topic","url":"\/documentation\/facecapture\/facecaptureerror\/helpanchor"},"doc://FaceCapture/documentation/FaceCapture/FaceCaptureError/recoverySuggestion":{"role":"symbol","title":"recoverySuggestion","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"recoverySuggestion"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}],"abstract":[],"identifier":"doc:\/\/FaceCapture\/documentation\/FaceCapture\/FaceCaptureError\/recoverySuggestion","kind":"symbol","type":"topic","url":"\/documentation\/facecapture\/facecaptureerror\/recoverysuggestion"},"doc://FaceCapture/documentation/FaceCapture/FaceCaptureError/failureReason":{"role":"symbol","title":"failureReason","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"failureReason"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}],"abstract":[],"identifier":"doc:\/\/FaceCapture\/documentation\/FaceCapture\/FaceCaptureError\/failureReason","kind":"symbol","type":"topic","url":"\/documentation\/facecapture\/facecaptureerror\/failurereason"},"doc://FaceCapture/documentation/FaceCapture/FaceCaptureError":{"role":"symbol","title":"FaceCaptureError","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"FaceCaptureError"}],"abstract":[{"type":"text","text":"Error thrown by the face capture library"}],"identifier":"doc:\/\/FaceCapture\/documentation\/FaceCapture\/FaceCaptureError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"FaceCaptureError"}],"url":"\/documentation\/facecapture\/facecaptureerror"}}}